test/functional/Java8andUp/playlist.xml

```diff
@@ -985,6 +985,7 @@
 		<variations>
 			<variation>NoOptions</variation>
 			<variation>-XX:RecreateClassfileOnload</variation>
+			<variation>-XX:+CompactStrings</variation>
```

Comment:
This is not really ideal either. The only subset we really want to run is the java.lang tests with compact strings. Now we can achieve this by extracting the `JCL_TEST_Java-Lang` suite into a new `<test>`, however there are 3 different tests already related to `JCL_TEST_Java-Lang` which are:

1. JCL_Test
2. JCL_Test_openj9_none_SCC
3. JCL_Test_SE80

So if I wanted to extract `JCL_TEST_Java-Lang` into a new `<test>` then I would have to make 3 of them which would put us up to a total of 6. This quickly gets out of control. This is why for (human) sanity I put it as a new variation here. Though this does increase build runtime. @llxia / @smlambert any thoughts on which way we should lean here? Your input is very valuable.

Reply:
@llxia @smlambert do you have any response?

Reply:
I would leave it as you have it as it only costs 1 minute to avoid ballooning to 6 different test cases.  I can not remember why JCL_Test_openj9_none_SCC exists as a separate case, or if it can/should be collapsed into case 1.